src/configurations/destinations/mp/schema.json

```diff
@@ -1,7 +1,7 @@
 {
   "configSchema": {
     "$schema": "http://json-schema.org/draft-07/schema#",
-    "required": ["token"],
+    "required": ["token", "dataResidency", "identityMergeApi"],
```

Comment:
As you're making these two parameters required, we need to check if there are any destinations with undefined values and might need to migrate them

Reply:
These are not new fields. 
schema generator script suggested to add the fields in schema.json